File: Python/XuatSo01.py
n, m = map(int, input().split())

a = max(n, m)
b = min(n, m)

if n == m:
    print("Gia tri cua n va m bang nhau", end="")
else:
    for i in range(a, b-1, -1):
        print(i, end="")
        if i != b: print(end=" ")
    

File: C++/XuatSo01.cpp
#include <iostream>
using namespace std;

int main() {
    int n, m; cin >> n >> m;
    int a = max(n, m);
    int b = min(n, m);

    if (a == b) {
        cout << "Gia tri cua n va m bang nhau";
    } else {
        for (int i = a; i >= b; i--) {
            cout << i;
            if (i != b) cout << " ";
        }
    }
    return 0;
}